### ISSUE TITLE
CVparam error on importing mzid into skyline

### ISSUE THREAD

**Issue by 1Moe** [compomics/peptide-shaker#185]
Hi Harald and Marc,

I had an issue opening a mzid export from peptideshaker in skyline.
Skyline complained about CV parameters associated with CH4OS (3 CVparams in total for CH4OS).

Versions I used:
Searchgui: 2.9.0
PeptideShaker: 1.11.0
Pwiz (64bit): 3.0.9248 
SkylineDaily: 3.5.1.9829

I ended up deleting lines from the mzid file that contained the CH4OS params and then skyline would load the truncated mzid fine.

I'm not sure if this is a SearchGui/Peptideshaker issue or a pwiz or a skyline issue but thought I'd let you know anyway. Below is the skylines error message

cheers :)
Moe

System.IO.IOException: ERROR: Invalid cvParam accession "1002694"
ERROR: 
ERROR: reading file SISPeptides2.mzid

   at pwiz.Common.SystemUtil.ProcessRunner.Run(ProcessStartInfo psi, String stdin, IProgressMonitor progress, IProgressStatus& status, TextWriter writer) in c:\proj\pwiz_x64\pwiz_tools\Shared\Common\SystemUtil\ProcessRunner.cs:line 59
   at pwiz.BiblioSpec.BlibBuild.BuildLibrary(LibraryBuildAction libraryBuildAction, IProgressMonitor progressMonitor, IProgressStatus& status, String[]& ambiguous) in c:\proj\pwiz_x64\pwiz_tools\Shared\BiblioSpec\BlibBuild.cs:line 171
   at pwiz.Skyline.Model.Lib.BiblioSpecLiteBuilder.BuildLibrary(IProgressMonitor progress) in c:\proj\pwiz_x64\pwiz_tools\Skyline\Model\Lib\BiblioSpecLiteBuilder.cs:line 121

**Comment by hbarsnes** [comment]
Hi Moe,

Thanks for telling us about this. Actually, I was made aware of an issue with the way in which we annotate fragment ion neutral losses yesterday and I'm already working on a fix. Will release a new version very soon that will hopefully also fix the problems with importing into Skyline.

BTW, do you know if Skyline uses the fragment ion annotation from the mzid file at all? If not, one option would be to make it up to the user to decide if this information should be included in the PeptideShaker mzid file or not? Would make the file a bit smaller and maybe make the export slightly faster as well. We already have this option implemented, we just need to make it available as an option to the users.

Best regards,
Harald

**Comment by hbarsnes** [comment]
Ho Moe,

Can you try loading the mzid file exported in this beta version in Skyline?

https://www.dropbox.com/s/27iy1saunnmursh/PeptideShaker-1.11.1-beta.zip?dl=0

At least I'm able to load the mzid export in Skyline from the PeptideShaker example dataset for this version, which was not case for the previous one.

Best regards,
Harald

**Comment by 1Moe** [comment]
Hi Harald,

thanks for your fast reply! 
The beta version of peptideshaker you provided works like a charm and produces a mzid that imports without issues into skyline!!!

I also posted on the skyline forum where I was told you're already working on the fix :)
https://skyline.gs.washington.edu/labkey/announcements/home/support/thread.view?entityId=7dcc854d-1b2b-1034-87f8-da20258210fe&_anchor=26639#row:26639

P.s. sorry, I'm not exactly sure what information skyline actually uses from the mzid. Tried to ask this on the skyline forum too - maybe I was not specific enough...
https://skyline.gs.washington.edu/labkey/announcements/home/support/thread.view?entityId=7dcc873d-1b2b-1034-87f8-da20258210fe&_anchor=26640#row:26640

Thanks again for your quick help and for all the effort you guys are putting into the development of searchgui/peptideshaker!

Best wishes
Moe
